tests/tests_common_test.go

```diff
@@ -111,6 +111,53 @@ func expectRecreateAfterDelete(res *testResource) {
 	Expect(err).ToNot(HaveOccurred())
 }
 
+func sspOperatorReconcilingProperly() (sum int) {
+	operatorPods, operatorMetricsPort := operatorPodsWithMetricsPort()
+	for _, sspOperator := range operatorPods {
+		sum += intMetricValue("ssp_operator_reconciling_properly", operatorMetricsPort, &sspOperator)
+	}
+	return
+}
+
+func totalRestoredTemplatesCount() (sum int) {
```

Comment:
This function is created in first commit and then modified in the second one. Can you create it this way in the first commit?

Reply:
squashing